lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -1079,7 +1080,9 @@ sub save_edit :Chained('base') :PathPart('save') :Args(0) {
 
             my $value = $c->req->params->{value};
             my $autocommit = $c->req->params->{autocommit};
-            my $complat_user = $c->d->rs('User')->find({username => $value});
+            my $rs_user = $c->d->rs('User');
+            my $complat_user = $rs_user->find({username => $value});
+            ($complat_user = $rs_user->find_by_github_login($value)) unless $complat_user;
```

Comment:
The above two lines are more simply written:
```Perl
my $complat_user = $rs_user->find({username => $value}) || $rs_user->find_by_github_login($value));
```
You're basically saying `$complat_user` is one or the other.